### DISCUSSION TITLE
Austin not working  - exits with exit code 64

### DISCUSSION THREAD

**Discussion by asafhaddad** [P403n1x87/austin#168] [Q&A]
Hi,

I'm trying Austin on VS code for the first time.
As a trial I have this simple script:

```
import numpy as np

# multiply two big random matrices
for i in range(10000):
    a = np.random.rand(1000,1000)
    b = np.random.rand(1000,1000)
    c = np.dot(a,b)
```

I'm running inside a Docker container in a Linux server.
When trying to run Austin from the command pallete (i.e. ctrl+shift+p --> 'Profile with Austin') and I get the following:

```
*  Executing task: austin: profile /workspace/src/repo/ROS/src/stat_bag_run.py 

Starting Profiler in /workspace/src.
Running 'austin' with args '-i 100 --pipe -s /bin/python3 /workspace/src/repo/ROS/src/stat_bag_run.py'.
austin process exited with code 64

 *  The terminal process failed to launch (exit code: 64). 
 *  Terminal will be reused by tasks, press any key to close it. 
```
Any idea on what's wrong?

Thanks!

**Comment by P403n1x87** [comment]
@asafhaddad that's quite a complex scenario that you're describing :slightly_smiling_face: . Can I get some more details from you to clarify a bit what is happening? Are you using the remote feature from VS Code to code directly into the container? Is Austin available from within the container and on the `PATH`? I haven't tested Austin in this scenario, but my guess is that VS Code is trying to launch Austin from the host system, using the path of the Python interpreter on the host.
